x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/action/filter/ShardBulkInferenceActionFilterTests.java

```diff
@@ -344,6 +344,9 @@ private static BulkItemRequest[] randomBulkItemRequest(
             }
 
             SemanticTextField semanticTextField;
+            // The model is not field aware and that is why we are skipping the embedding generation process for existing values.
+            // This prevents a situation where embeddings in the expected docMap do not match those in the model, which could happen if
+            // embeddings were overwritten.
```

Comment:
Do you think this comment makes sense @Mikep86?

Reply:
Looks good to me, thanks for the iterations!